cmd/cmd/logFileShow.go

```diff
@@ -0,0 +1,105 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package cmd
+
+import (
+    "fmt"
+    "github.com/spf13/cobra"
+    "github.com/wso2/product-mi-tooling/cmd/utils"
+    "github.com/wso2/product-mi-tooling/cmd/utils/artifactUtils"
+    "os"
+)
+
+var logfileName string
+
+// Show Logs command related usage info
+const showLogsCmdLiteral = "show"
+const showLogsCmdShortDesc = "Get information about log files"
```

Comment:
Shall we change this to 'Show available log files'?

Reply:
+1 fixed